ports/nodesoup/vcpkg.json

```
@@ -0,0 +1,16 @@
+{
+  "name": "nodesoup",
+  "version": "0.1.0",
```

```suggestion
  "version-date": "2023-06-12",
```
The upstream does not have any Releases and Tags. In this case, `version-date` is recommended.